.travis.yml

```diff
@@ -118,25 +118,10 @@ jobs:
 script:
   - export RUN_JOB=1
   - ./.ci/travis/travis.sh init-m2-repo
-  - |
-    if [[ $RUN_JOB == 1 ]]; then
-      echo "eval of CMD is starting";
-      echo "CMD=$CMD";
-      eval $CMD;
-      echo "eval of CMD is completed";
-    fi
+  - ./.ci/travis/travis.sh run-command "$CMD"
```

Comment:
I couldn't find an issue but I am sure this has been talked about. We discussed before splitting shell scripts on the activity they do instead of the CI as we had to move around CIs a few times already. For example, having a shell script for all regression so everything for that activity is all in one place instead of split if 1 CI does a few and another CI does others.

Doesn't this change sort of take us back and make doing that a bit more complicated?

Reply:
this is complicated.
I do not know why ... I still try to think that CI specific config contains commands to run. But Travis force us to keep all in travis.sh

```
    - jdk: openjdk8
      env:
        - DESC="NoErrorTest - Spring Cloud GCP"
        - CMD="./.ci/travis.sh no-error-spring-cloud-gcp"
        - USE_MAVEN_REPO="true"
```

./.ci/travis.sh:
```
no-error-spring-cloud-gcp)
  mvn -e --no-transfer-progress clean install -Pno-validations
  ./.validation.sh no-error-spring-cloud-gcp
```

may be we should use above pattern.
So CI config and CI specific shell remove all CI specifics, and under them there is validaiton.sh that try to CI agnostic.